rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
```

Comment:
- [ ] This is a close one as well but to achieve the 'belongs-to' association the `Rental` object should add itself to the rentals attribute found in the person and book objects as shown below:
```rb
  book.rentals << self
  person.rentals << self
```